bin-example/generation/src/main/scala/software/uncharted/salt/examples/bin/Main.scala

```diff
@@ -49,10 +49,10 @@ object Main {
     val sparkSession = SparkSession.builder.appName("salt-bin-example").getOrCreate()
     val sc = sparkSession.sparkContext
 
-    sparkSession.read.format("com.databricks.spark.csv")
+    sparkSession.read.format("csv")
```

Comment:
you can do `sparkSession.read.option(...).option(...).csv("sync/example.csv")`
https://spark.apache.org/docs/2.0.0/api/scala/index.html#org.apache.spark.sql.DataFrameReader